feature-sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/Sessions.kt

```diff
@@ -82,11 +84,13 @@ fun Sessions(
 ) {
     val scheduleState = uiModel.scheduleState
     val pagerState = rememberPagerState()
+    val listStates = DroidKaigi2022Day.values().map { rememberLazyListState() } .toList()
```

Comment:
This is very minor, but how about changing the name to sessionsListListStates? The reason is that it could be mistaken for Timetable's.

Reply:
You are right. I have addressed your point. [07ce675](https://github.com/DroidKaigi/conference-app-2022/pull/206/commits/07ce675915159c71fddb84c668097d9235eca274)

This commit also include bug fix (This feature should not work on TimeTable tab.).